recipes/nuraft/all/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.11)
+project(nuraft)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(Boost REQUIRED)
+find_package(OpenSSL REQUIRED)
+
+add_compile_options("-DUSE_BOOST_ASIO=1")
```

Comment:
I would recommend using CMakeToolchain instead. So you no longer need this cmake wrapper: https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmaketoolchain.html#preprocessor-definitions